python/modules/jetCorr.py

```diff
@@ -0,0 +1,173 @@
+###
+# Useful CMStalk thread on how implementing jet corrections: https://cms-talk.web.cern.ch/t/jes-for-2022-re-reco-cde-and-prompt-fg/32873/3
+# Minimal demo provided from JME POG: https://github.com/cms-jet/JECDatabase/blob/master/scripts/JERC2JSON/minimalDemo.py
+# TODO:
+# - Implementing JES uncertainties. TBD which scheme we want to implement
+###
+
+from __future__ import print_function
+from PhysicsTools.NanoAODTools.postprocessing.framework.eventloop import Module
+from PhysicsTools.NanoAODTools.postprocessing.framework.datamodel import Collection
+import os
+import random
```

Comment:
Looks like that "random" is not used (the smearing is done inside correctionlib) - I would remove this.

Reply:
Other than this, everything looks fine to me, thanks @AlessandroTarabini !

Reply:
Done ;)